### ISSUE TITLE
support for Micro QR Code

### ISSUE THREAD

**Issue by Tibbel** [nayuki/QR-Code-generator#108]
Is it possible to use it to create "Micro QR Codes"?
Or add support for it?

Thanks

**Comment by nayuki** [comment]
No. See this identical past topic: https://github.com/nayuki/QR-Code-generator/issues/98

**Comment by Tibbel** [comment]
Okay, thanks
